.pre-push

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -x
+set -e
+
+make check-go-format
```

Comment:
@xcoulon @hectorj2f can you please try running these commands:

```sh
# Delete any old formatting violation that was found (just to be safe)
rm -f /tmp/gofmt-errors
make check-go-format && echo "ALL GOOD"
```

In d3396ce I've introduced what files we want to format and check.